car/car.ino

```diff
@@ -143,8 +144,8 @@ void handleInput() { //handle serial input (String!!)
 void obstacleAvoidance(){
   int leftDistance = left.readRangeContinuousMillimeters();
```

Comment:
Row 145-147:
rightDistance and leftDistance get value from the sensors. We tested and printed the values. 

- leftDistance was always >1000 (mm) 
- rightDistance was always == 0 (cm) 

That means that leftDistance was always larger than LEFT_DIST (500 mm) and rightDistance was always == 0 following the logigc of the if-statment. 

We also tested the if-statements by printing letters for each. They work as intended meaning if there is no object on either side of the statement it goes into the first if, if on the left side it goes into the second if, if on the right side it goes into the third and if objects on all sides it goes into the else. 

We have located that the problem should be in the if-statement, somewhere, but we can not figure out what might couse the problem.